src/platform_impl/linux/util.rs

```diff
@@ -27,3 +28,24 @@ pub fn cursor_position(is_wayland: bool) -> Result<PhysicalPosition<f64>, Extern
       .ok_or(ExternalError::Os(os_error!(super::OsError)))?
   }
 }
+
+pub fn is_unity() -> bool {
+  if let Ok(child) = Command::new("echo")
+    .arg("$XDG_CURRENT_DESKTOP")
+    .stdout(Stdio::piped())
+    .spawn()
```

Comment:
```suggestion
  if let Ok(child) = Command::new("sh")
    .arg("-c")
    .arg("echo -n $XDG_CURRENT_DESKTOP")
    .stdout(Stdio::piped())
    .spawn()
```

`echo` is built-in the `sh` so you need to call it from `sh`

`echo -n` to trim the newline that return from the `echo` otherwise the return will be like `Unity\n`